daisy_workflows/build-publish/sqlserver/sql-2012-enterprise-windows-2012-r2-dc.wf.json

```diff
@@ -0,0 +1,76 @@
+{
+  "Name": "sql-2012-enterprise-windows-2012-r2-dc",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-b",
+  "GCSPath": "gs://gce-image-build-bucket/daisy/${USERNAME}",
+  "Vars": {
+    "build_date": {
+      "Value": "${TIMESTAMP}",
+      "Description": "Build datestamp used to version the image."
+    },
+    "gcs_url": {
+      "Required": true,
+      "Description": "The GCS url that the image raw file exported to."
+    },
+    "gcs_internal_url": {
+      "Required": true,
+      "Description": "The GCS url that the exported image is saved to for internal use."
+    },
+    "workflow_root": {
+      "Value": "/workflows",
+      "Description": "Root of github workflows, defaults to /workflows in the container."
+    },
+    "install_disk": "disk-install",
+    "publish_project": "${PROJECT}",
+    "sql_server_media": {
+      "Required": true,
+      "Description": "GCS or local path to the SQL Server installer media"
+    },
+    "source_image_project": {
+      "Value": "windows-cloud",
+      "Description": "Project to source base image from."
+    },
+    "ssms_exe": {
+      "Required": true,
+      "Description": "GCS or local path to SSMS installer"
+    },
+    "timeout": {
+      "Value": "2h",
+      "Description": "The timeout to set for the image build."
+    }
+  },
+  "Steps": {
+    "build": {
+      "Timeout": "4h",
+      "IncludeWorkflow": {
+        "Path": "${workflow_root}/image_build/sql-2012-enterprise-windows-2012-r2-dc.wf.json",
+        "Vars": {
+          "build_date": "${TIMESTAMP}",
+          "install_disk": "disk-install",
+          "publish_project": "${publish_project}",
+          "sql_server_media": "${sql_server_media}",
+          "source_image_project": "${source_image_project}",
+          "ssms_exe": "${ssms_exe}",
+          "timeout": "2h"
+        }
+      }
+    },
+    "sqlserver-export": {
+      "Timeout": "60m",
+      "IncludeWorkflow": {
```

Comment:
I don't think we need our own sql-server export process. Since there is a single file output we can use the common export/disk_export.wf.json",

ex:
      "IncludeWorkflow": {
        "Path": "${workflow_root}/export/disk_export.wf.json",
        "Vars": {
          "source_disk": "${install_disk}",
          "destination": "${gcs_path}/${image_prefix}-v${build_date}/root.tar.gz"
        }
      }